auth/client-api.md

```diff
@@ -18,9 +18,12 @@ abstract class Client {
 
   // query all pending requests
   public abstract getPendingRequests(): Promise<Record<number, PendingRequest>>;
-
-  // query cached response matching id
-  public abstract getResponse(params: { id: number }): Promise<Response>;
+  
+  // querry pairings
+  public abstract getPairings(): Promise<Array<Pairing>>;
+  
+  // for either to ping a peer in a pairing
+  public abstract ping(params: { topic: string; }): Promise<void>;
```

Comment:
If we let users to get list of pairings should we also allow them to delete one? Imo It's be cool

Reply:
make sense to me, I will add it!